Gjhnsn/react-challenge-project-sep-trial
# Update dependencies

## Changes
_List Changes Introduced by this PR_
1. Updated dependencies in application and server

## Purpose
To avoid security risks due to outdated securities.

## Approach
Ran `npm outdated` to view outdated dependencies.
Updated to latest versions.

## Testing Steps
Pull down project, observe dependency versions.
Run `npm outdated` to check

_If this closes an issue, name it here. If it doesn't, remove this line_
Closes #7
